src/redux-query-sync.js

```diff
@@ -71,7 +71,7 @@ function ReduxQuerySync({
         // For each parameter value that changed, call the corresponding action.
         Object.keys(queryValues).forEach(param => {
             const value = queryValues[param]
-            if (value !== lastQueryValues[param]) {
+            if (value !== lastQueryValues[param] && value !== undefined) {
```

Comment:
I'm not sure I understand the problem this is supposed to solve, could you explain it more precisely? On the initial page load (with `initialTruth: 'location'`, I assume?), the values of `lastQueryValues` would be `undefined`, right? So how would this check help?

In any case, if there is some problem with initial values, the solution may rather be to initialise `lastQueryValues` differently, or make some other changes to how things start off. Adding this check here could break behaviour in other situations.